core/src/elo/elo-connector/schemas/index.ts

```diff
@@ -1,3 +1,4 @@
+export * from "./AddPlayer.schema";
```

Comment:
Make name consistent with what is in the file.
```suggestion
export * from "./AddPlayers.schema";
```